expression/builtin_miscellaneous_vec_test.go

```diff
@@ -14,11 +14,15 @@
 package expression
 
 import (
+	"sync/atomic"
 	"testing"
+	"time"
 
 	. "github.com/pingcap/check"
 	"github.com/pingcap/parser/ast"
 	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/chunk"
+	"github.com/pingcap/tidb/util/mock"
 )
 
 var vecBuiltinMiscellaneousCases = map[string][]vecExprBenchCase{
```

Comment:
A comparison test is also necessary to be present here in `ast.Sleep`.

Reply:
Is it possible to enable non-strict in the comparison test? If we only use test cases greater than zero, it won't cover any branches.

Reply:
Because the sleep function will sleep, time to finish `TestVectorizedBuiltinMiscellaneous` increases a lot.
```
tidb $ go test ./expression -check.f TestVectorizedBuiltinMiscellaneous
ok      github.com/pingcap/tidb/expression      227.923s
```